tests/driver_lis2dh12/Makefile

```diff
@@ -1,10 +1,13 @@
 include ../Makefile.tests_common
 
-# as default we use the SPI mode for now, as the I2C mode is not supported, yet
-DRIVER ?= lis2dh12_spi
+# use lis2dh12_spi for SPI-Mode and lis2dh12_i2c for I2C-Mode
+DRIVER ?= lis2dh12_i2c
```

Comment:
For what reasons did you change the default driver module?

Reply:
There is no good reason, I only tested it with a device using I2C.
I can change the default back to SPI.

Reply:
> okay I will change this back to SPI. I thought it does not matter, because you marked this as resolved.
> Sry =)

Yeah, I marked it as resolved because I though it is changed backed :sunglasses: sorry my fault.